corehq/warehouse/models/meta.py

```diff
@@ -3,8 +3,6 @@
 
 
 class Batch(models.Model):
-    batch_id = models.UUIDField(unique=True, db_index=True, primary_key=True)
-
```

Comment:
we should probably make all batch id foreign key columns be ints now too. not sure if we'll be able to use an actual foreign key if any facts get sharded. but if we ever have the capability to replicate certain tables across all shards, the batch table would be one to do that with

Reply:
do you know how to do this in django? All of the column specs in the migrations just say foreign key (it must determine the column type automatically). Perhaps adding a new migration with the same specification will force it to redetermine what type it is.

Reply:
hmm, when you run the migration in this pr does it update the foreign key columns too? if not a manual migration might be needed

Reply:
looks like i never responded to this comment but I have added migrations to do that in future commits